static/js/custom.js

```diff
@@ -88,7 +87,10 @@ function addClass(el, cl) {
     if(el) el.className = el.className + " " + cl;
 }
 
-function incrCommentVote(e) {
-    var node = e.parentNode.parentNode.childNodes[0];
-    node.data = Number(node.data) + 1;
+function handleLikeOrVote(e) {
+    // FIXME: this is a race condition: if we wait for 0 ms, the page
+    // will usually be reloaded before the POST request updating the
+    // score can be processed.
+    // FIXME: we are required to avoid reload!
+    setTimeout(function() { document.location.reload(true); }, 50);
```

Comment:
> @fisx why did you removed the previous version?

sorry (:

my solution wasn't as successful as i hoped, but it has the nice effect on chrome that it looks almost like it works, even when clicking erratically on both many times.  your solution was not supporting the case where the user changed their mind.

anyway we'll need something more sophisticated.  probably involving ajax.